src/moveTo.js

```diff
@@ -121,7 +121,7 @@ const MoveTo = (() => {
    * @param  {object} options Custom options
    */
   MoveTo.prototype.move = function(target, options = {}) {
-    if (target !== 0 && !target) {
```

Comment:
`target !== 0` should not be deleted here, otherwise if target is `0` the scroll animation will not work.